### ISSUE TITLE
test hyperparameters

### ISSUE THREAD

**Issue by JonasEinig** [spaethju/skin-cancer-detection#15]
- optimizer (Adam/ MomentumOptimizer/SGD/AdaMax/RMSProp)
- learning rate (Adam: 0.001, Momentum and SGD 10^-9 ) 
- batch size (1,2,3,4,5,6)
